data/quarg missions.txt

```diff
@@ -13,6 +13,7 @@ mission "First Contact: Quarg"
 	source
 		attributes "quarg"
 	on offer
+		log "Visited a Quarg spaceport. Spoke with one of them briefly."
```

Comment:
Yeah, I was bothered with how to best branch this. If you ask questions, you _really_ want the log to contain the info you get. But since you can not ask anything, it's hard to write a good log entry.

A good idea might be to set conditions in the branches and use them to run some invisible missions. It's not ideal, but it'd work.